pkg/backend/httpstate/stack.go

```diff
@@ -98,6 +98,15 @@ func (c cloudBackendReference) FullyQualifiedName() tokens.QName {
 	return tokens.IntoQName(fmt.Sprintf("%v/%v/%v", c.owner, c.project, c.name.String()))
 }
 
+// Returns configured default org, if configured..
+// If unset, will fallback to requesting default org from the backend, which may involve an additional API call.
```

Comment:
Can't we just assert it's always set? It's always built by the http backend we can just make sure we always set it? It's probably always more efficient to just set it at construction than to defer it till it's stringified (it nearly _always_ gets stringified)

Reply:
Let me double check all the call sites!